cmd/variables.go

```diff
@@ -7,16 +7,18 @@ import (
 )
 
 var (
-	defaultNetwork     = common.GetEnv("NETWORK", "")
-	defaultBeaconURIs  = common.GetSliceEnv("BEACON_URIS", []string{"http://localhost:3500"})
-	defaultRedisURI    = common.GetEnv("REDIS_URI", "localhost:6379")
-	defaultPostgresDSN = common.GetEnv("POSTGRES_DSN", "")
-	defaultLogJSON     = os.Getenv("LOG_JSON") != ""
-	defaultLogLevel    = common.GetEnv("LOG_LEVEL", "info")
+	defaultNetwork            = common.GetEnv("NETWORK", "")
+	defaultBeaconURIs         = common.GetSliceEnv("BEACON_URIS", []string{"http://localhost:3500"})
+	defaultRedisURI           = common.GetEnv("REDIS_URI", "localhost:6379")
+	defaultPostgresDSN        = common.GetEnv("POSTGRES_DSN", "")
+	defaultMemcachedEndpoints = common.GetSliceEnv("MEMCACHED_ENDPOINTS", nil)
```

Comment:
please use the term URI as with beacon, redis, for conciseness:

```suggestion
	defaultMemcachedURIs = common.GetSliceEnv("MEMCACHED_URIS", nil)
```

Reply:
sure, done!

Reply:
please feel free to un-resolve if I missed anything